tests/http_proxy_tests.py

```diff
@@ -0,0 +1,183 @@
+from unittest.mock import MagicMock
+from pathlib import Path
+from copy import deepcopy
+
+from urllib.parse import urlencode
+from flask import json, url_for
+
+from superdesk import __version__ as superdesk_version
+from superdesk.tests import TestCase, setup_auth_user
+from superdesk.http_proxy import HTTPProxy, register_http_proxy
+
+
+class HttpProxyTestCase(TestCase):
+    proxy = HTTPProxy("test_proxy", internal_url="test/proxy", external_url="http://localhost:5001/api")
```

Comment:
I wonder if we need the ability to specify that `internal_url`, maybe using one route with: `/proxy/<endpoint_name>/<path>` would be enough, otherwise it will be possible to override internal apis